Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -161,12 +161,19 @@ We use primes in the structure names as we will reformulate them below (without
 /-- A structure groupoid is a set of partial homeomorphisms of a topological space stable under
 composition and inverse. They appear in the definition of the smoothness class of a manifold. -/
 structure StructureGroupoid (H : Type u) [TopologicalSpace H] where
+  /-- Members of the structure groupoids are partial homeomorphisms. -/
   members : Set (PartialHomeomorph H H)
+  /-- Structure groupoids are stable under composition. -/
   trans' : ∀ e e' : PartialHomeomorph H H, e ∈ members → e' ∈ members → e ≫ₕ e' ∈ members
+  /-- Structure groupoids are stable under inverse. -/
   symm' : ∀ e : PartialHomeomorph H H, e ∈ members → e.symm ∈ members
+  /-- The identity is in the structure groupoid. -/
   id_mem' : PartialHomeomorph.refl H ∈ members
+  /-- If the restriction of a partial homeomorphism to some open set around every point in its
```

Comment:
I think this docstring hard to parse: is this better?
"Let `e` be a partial homeomorphism. If for every `x \in e.source`, the restriction of `e` to some open set around `x` lies in the groupoid, then `e` lies in the groupoid."